internal/lifecycle/readme.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lifecycle
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/GoogleCloudPlatform/serverless-sample-tester/internal/util"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+)
+
+// The tag that should appear immediately before code blocks in a README to indicate that the enclosed commands
+// are to be used by this program for building and deploying the sample.
+var codeTag = "sst-run-linuxmacos"
```

Comment:
I think it would be the same for Unix. But I wasn't not sure how the `exec.Command` and associated go interfaces.

One thing that concerns me specifically is that os.ExpandEnv seems to only expand variables with the `$VAR` or `${VAR}` syntax. Whereas it appears that Windows Command Prompt users can expand variables using the `%VAR%` syntax.

Reply:
Might want to just focus on bash syntax for now, since Windows syntax has a couple of variations. Windows CMD prompt expands environment variables with ```%varname%``` syntax, and PowerShell (which is more commonly used these days) expands them with ```$env:varname``` syntax.